src/properties/font.rs

```diff
@@ -908,6 +912,42 @@ impl<'i> PropertyHandler<'i> for FontHandler<'i> {
   }
 }
 
+/// [`system-ui`](https://www.w3.org/TR/css-fonts-4/#system-ui-def) is a special generic font family
+/// It is platform dependent but if not supported by the target will simply be ignored
+/// This list is an attempt at providing that support
+#[inline]
+fn compatible_font_family(family: Option<Vec<FontFamily>>, is_supported: bool) -> Option<Vec<FontFamily>> {
+  let system_ui = &FontFamily::Generic(GenericFontFamily::SystemUI);
+
+  let default_system_fonts: [FontFamily; 8] = [
```

Comment:
With the suggestion below, this could be a const of strings instead:

```rust
const DEFAULT_SYSTEM_FONTS: &[&str] = &[
  // #1: Supported as the -apple-system value (only on Mac)
  "AppleSystem",
  // #2: Supported as the 'BlinkMacSystemFont' value (only on Mac)
  "BlinkMacSystemFont",
  "Segoe UI",  // Windows >= Vista
  "Roboto",    // Android >= 4
  "Noto Sans", // Plasma >= 5.5
  "Ubuntu",    // Ubuntu >= 10.10
  "Cantarell", // GNOME >= 3
  "Helvetica Neue",
];
```

Reply:
done. I tried to extract it before but by keeping the `FontFamily` and the compiler told me that `CowArcStr` could not be in a static context. looks better indeed